README.md

```diff
@@ -1,3 +1,5 @@
+<img src="https://raw.githubusercontent.com/thoughtbot/Swish/gh-pages/swish-logo-v4.jpg" width="500">
+
```

Comment:
We've been keeping these separate (see [Argo](https://github.com/thoughtbot/argo))